CMakeLists.txt

```diff
@@ -189,7 +189,7 @@ if(SDL2_FOUND AND SDL2_mixer_FOUND)
 	message(STATUS "Using SDL2 ${SDL2_VERSION} and SDL2_mixer ${SDL2_mixer_VERSION}.")
 endif()
 
-target_link_libraries(alex4 miniz iniparser SDL2::SDL2 SDL2_mixer::SDL2_mixer)
```

Comment:
I think this is wrong, the imported libraries need the prefixes.

Reply:
Can you check the cmake files provided by sdl2_mixer on your system? Might have been fixed upstream in the meanwhile. Here with `2.8.0` in `/usr/lib/cmake/SDL2_mixer/sdl2_mixer-config.cmake` it is correct.

If nothing else helps, we could create an `ALIAS` for it.

Reply:
Maybe this is broken in sdl2_mixer build system (using autotools vs. cmake).

Reply:
Well, I think this is still correct though. https://github.com/libsdl-org/SDL_mixer/blob/6229486690ee1f799aa01c1235fb39491d0fbc9d/sdl2_mixer-config.cmake.in#L116